app/api/__init__.py

```diff
@@ -73,10 +74,11 @@
 api.route(EventRelationship, 'event_tax', '/events/<int:id>/relationships/tax')
 api.route(EventRelationship, 'event_event_invoice', '/events/<int:id>/relationships/event-invoices')
 api.route(EventRelationship, 'event_discount_code', '/events/<int:id>/relationships/discount-codes')
+api.route(EventRelationship, 'event_session', '/events/<int:id>/relationships/sessions')
 
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations',
-          '/events/<int:id>/microlocations', '/sessions/<int:id>/microlocations')
+          '/events/<int:id>/microlocations')
```

Comment:
I understand we are doing this in many places. But I'm not sure why we are not making convention for 
```/events/<event_id>/microlocations``` 
In case we get more than one entries here like we have in ```SessionList``` route then we cannot just say `id` instead we will have ```event_id``` so why not use it everywhere even for a single entry. And also at many places, we are using ```event_id``` even for the single entry. So as per my suggestion, we should change all entries to ```event_id``` ( related type )